docs/content/modules/carts-and-checkout/backend/add-payment-provider.md

```diff
@@ -1,158 +1,167 @@
+description: 'Learn how to create a payment processor in the Medusa server. This guide explains the different methods available in a payment processor.'
 addHowToData: true
 ---
 
-# How to Create a Payment Provider
+# How to Create a Payment Processor
 
-In this document, you’ll learn how to add a Payment Provider to your Medusa backend. If you’re unfamiliar with the Payment architecture in Medusa, make sure to check out the [overview](../payment.md) first.
+In this document, you’ll learn how to create a Payment Processor in your Medusa backend. If you’re unfamiliar with the Payment architecture in Medusa, make sure to check out the [overview](../payment.md) first.
+
+:::note
+
+Before v1.8 of Medusa, this guide explained how to create a payment provider. Payment Providers are now considered legacy and are deprecated. Moving forward, it's recommended to create a Payment Processor that implements the Payment Processor API.
+
+:::
 
 ## Overview
 
-A Payment Provider is the payment method used to authorize, capture, and refund payment, among other actions. An example of a Payment Provider is Stripe.
+A Payment Processor is the payment method used to authorize, capture, and refund payment, among other actions. An example of a Payment Processor is Stripe.
 
 By default, Medusa has a [manual payment provider](https://github.com/medusajs/medusa/tree/master/packages/medusa-payment-manual) that has minimal implementation. It can be synonymous with a Cash on Delivery payment method. It allows store operators to manage the payment themselves but still keep track of its different stages on Medusa.
 
-Adding a Payment Provider is as simple as creating a [service](../../../development/services/create-service.md) file in `src/services`. A Payment Provider is essentially a service that extends `AbstractPaymentService` from the core Medusa package `@medusajs/medusa`.
+Adding a Payment Processor is as simple as creating a [service](../../../development/services/create-service.md) file in `src/services`. A Payment Processor is essentially a service that extends `AbstractPaymentProcessor` from the core Medusa package `@medusajs/medusa`.
 
-Payment Provider Services must have a static property `identifier`. It's the name that will be used to install and refer to the Payment Provider in the Medusa backend.
+Payment Processor Services must have a static property `identifier`. It's the name that will be used to install and refer to the Payment Processor in the Medusa backend.
 
 :::tip
 
-Payment Providers are loaded and installed at the backend startup.
+Payment Processors are loaded and installed at the server startup. If not already saved, they're saved in the database and are represented by the `PaymentProvider` entity.
 
 :::
 
-The Payment Provider service is also required to implement the following methods:
+The Payment Processor is also required to implement the following methods:
```

Comment:
Not sure what you mean here. Do you mean that we should indicate that these belong to the interface? If so I don't think it's necessary to highlight here

Reply:
It is just to reference it as it is also documented in the interface but not the abstract class since it extend the interface. I will follow your choice :)